browser/ui/brave_browser_command_controller.cc

```diff
@@ -219,15 +226,29 @@ void BraveBrowserCommandController::InitBraveCommandState() {
   UpdateCommandForWebcompatReporter();
 #if BUILDFLAG(ENABLE_TOR)
   UpdateCommandForTor();
+#endif
+#if BUILDFLAG(ENABLE_AI_CHAT)
```

Comment:
Does this need to happen before the other commands or could we join it in to the other `#if BUILDFLAG`s on 237